doc/radosgw/frontends.rst

```diff
@@ -59,6 +59,17 @@ Options
 :Type: String
 :Default: None
 
+``tcp_nodelay``
+
+:Description: If set the socket option will disable Nagle's algorithm on 
+              the connection which means that packets will be sent as soon 
+              as possible instead of waiting for a full buffer or timeout to occur.
+              ``1    Disable Nagel's algorithm for all sockets.``
```

Comment:
hmm i'm not sure how to get it to render with line breaks here. maybe leave these last two lines out, and change the first sentence to `If set to ``1``, the socket option ...`?

if you care to play around with the syntax, there's a `ceph/admin/build-doc` script to build the docs locally and a `ceph/admin/serve-doc` to serve them over http

Reply:
![tcp_nodelay_section](https://user-images.githubusercontent.com/48631098/54817546-ea8b0080-4c9f-11e9-9c0d-59ba2ba38d3e.png)
Is it fine now? if yes I will commit it.

Thank you